datacenter/ucp/3.0/reference/cli/restore.md

```diff
@@ -45,25 +45,26 @@ backup file under /config/backup.tar, and run the restore command with the
 
 Notes:
 
-  * Run `uninstall-ucp` before attempting the restore operation on an
+  * Please run `uninstall-ucp` before attempting the restore operation on an
```

Comment:
I'd go without the Please. More imperative tone is best here. Don't be afraid to direct the reader.